script/validate_json.rb

```diff
@@ -22,7 +22,7 @@ module ExitCodes
 
 SupportedDifficulties = ["easy", "medium", "hard", "impossible"]
 SupportedEntryKeys = ["difficulty", "domains", "email", "email_body", "email_subject", "meta", "name", "notes", "url"]
-SupportedLanguageKeys = ["about", "contribute", "difficulty", "difficulty_easy", "difficulty_hard", "difficulty_impossible",
+SupportedLanguageKeys = ["about", "contribute", "defaultnote_easy", "defaultnote_email", "difficulty", "difficulty_easy", "difficulty_hard", "difficulty_impossible",
```

Comment:
I know it will be painful to reorder, but this was tidy as to not being longer than some column width (I'd guess 100 or 120)

Reply:
Alternatively, I'd also be fine with this list being reformatted to be one item per line, particularly since it's so long.

Reply:
I'll open a separate issue for that